replications/internal/queue_management.go

```diff
@@ -21,7 +21,7 @@ import (
 const (
 	scannerAdvanceInterval = 10 * time.Second
 	purgeInterval          = 60 * time.Second
-	defaultMaxAge          = 168 * time.Hour / time.Second
+	defaultMaxAge          = 168 * time.Hour
```

Comment:
I know this isn't your original code but can we change this to `7 * 24 * time.Hour // 1 week` just to make it clear what the intent is here?